tools/run_tests.py

```diff
@@ -19,28 +19,28 @@
 
 class CITester(object):
 
-    def __init__(self, dcos_url, github_label='integration'):
+    def __init__(self, dcos_url, github_label):
         self.__CLI_URL_TEMPLATE = 'https://downloads.dcos.io/binaries/cli/{}/x86-64/latest/{}'
         self.__dcos_url = dcos_url
         self.__sandbox_path = ''
-        self.__sandbox_env = {}
         self.__github_updater = github_update.GithubStatusUpdater('test:{}'.format(github_label))
 
 
-    def __get_cli_sandbox(self):
-        self.__sandbox_path = tempfile.mkdtemp(prefix='ci-test')
-        custom_env = os.environ.copy()
+    def __configure_cli_sandbox(self):
```

Comment:
I've just been using it to indicate 'no really this is private' -- I haven't really kept up with python idioms since 2.4 or so.